WordPress/UITestsFoundation/Globals.swift

```diff
@@ -31,6 +31,17 @@ public func pullToRefresh(app: XCUIApplication = XCUIApplication()) {
     top.press(forDuration: 0.01, thenDragTo: bottom)
 }
 
+public func waitAndTap( _ element: XCUIElement) {
+    var retries = 0
+    let maxRetries = 10
+    if element.waitForIsHittable(timeout: 10) {
```

Comment:
Sorry if this is an obvious question but I'm a little unsure how this `timeout` works, in this case, will the loop be trying every second for 10 seconds before timing out or is there a 10-second wait before a retry?

Reply:
Not an obvious question at all, @jostnes. Thanks for asking! The nesting of `if` and `while` made the code a bit confusing for me, too.

As far as I understand without actually running the code, `if element.waitFor...` will check that `isHittable = true` periodically, for a max of 10 seconds (see [the source](https://github.com/Automattic/XCUITestHelpers/blob/5179cb69d58b90761cc713bdee7740c4889d3295/Sources/XCUITestHelpers/XCUIElement%2BAsync.swift#L5-L16)). If it finds that `isHittable = true` it will start the loop, otherwise, the method will return.

If the loop starts, the UI tests will basically tap the element until it is no longer hittable, up to 10 times.

Reply:
Thanks for clarifying that @mokagio! That clears it up - it will try to tap the element a max of 10 times within 10 seconds.

Reply:
> it will try to tap the element a max of 10 times within 10 seconds.

Just to be clear: It will wait for the element to be hittable for up to 10 seconds, _then_ it will try to tap it up to 10 times.

There is no timeout for how long the "try to tap up to 10 times" should take.

Reply:
Ah okay, I'm glad I summarized my understanding so it can be corrected. The timeout is only for `element.waitForIsHittable(timeout: 10)` and not for the loop. 

I didn't see your suggested change below previously, it's clearer now 😄

Reply:
Thanks for taking a look, @jostnes !